### PR TITLE
agent: log nitfixes

### PR DESCRIPTION
**Pull request by sknat**
Signed-off-by: Nathan Skrzypczak <nathan.skrzypczak@gmail.com>

### CODE REVIEW

File: calico-vpp-agent/cni/network_vpp.go [hunk 1]
```diff
@@ -26,6 +28,10 @@ import (
 	"github.com/projectcalico/vpp-dataplane/vpplink/types"
 )
 
+type PodNSNotFoundErr struct{ msg string }
+
+func (e PodNSNotFoundErr) Error() string { return e.msg }
+
```
Comment (AloysAugustin) [inline calico-vpp-agent/cni/network_vpp.go:34]:
```suggestion
type PodNSNotFoundErr struct { 
    ns string
}

func (e PodNSNotFoundErr) Error() string {
    return fmt.Sprintf("Netns '%s' doesn't exist, skipping", e.ns)
}
```

File: calico-vpp-agent/cni/network_vpp.go [hunk 2]
```diff
@@ -81,7 +87,7 @@ func (s *Server) AddVppInterface(podSpec *storage.LocalPodSpec, doHostSideConf b
 
 	err = ns.IsNSorErr(podSpec.NetnsName)
 	if err != nil {
-		return vpplink.InvalidID, errors.Wrapf(err, "Netns '%s' doesn't exist, skipping", podSpec.NetnsName)
+		return vpplink.InvalidID, PodNSNotFoundErr{ fmt.Sprintf("Netns '%s' doesn't exist, skipping", podSpec.NetnsName) }
```
Comment (AloysAugustin) [inline calico-vpp-agent/cni/network_vpp.go:90]:
```suggestion
		return vpplink.InvalidID, PodNSNotFoundErr{podSpec.NetnsName}
```

File: calico-vpp-agent/watchers/peers_watcher.go [hunk 3]
```diff
@@ -160,7 +160,7 @@ func (w *PeerWatcher) WatchBGPPeers(t *tomb.Tomb) error {
 			}
 			switch event.Type {
 			case watch.EventType(api.WatchError):
-				w.log.Infof("peers watch returned an error")
+				w.log.Infof("peers watch returned, restarting...")
```
Comment (AloysAugustin) [inline calico-vpp-agent/watchers/peers_watcher.go:163]: The 3 "watch ended" logs could maybe be lowered to Debug level?

**Review by AloysAugustin — CHANGES_REQUESTED** [review]

File: calico-vpp-agent/cni/network_vpp.go (hunk 1 above)
Comment (sknat) [reply calico-vpp-agent/cni/network_vpp.go]: Yes, defintely 👍

**Pull request merged**